go.mod

```diff
@@ -82,11 +82,10 @@ require (
 	github.com/vektah/gqlparser/v2 v2.5.8
 	github.com/zenazn/goji v1.0.1
 	go.mongodb.org/mongo-driver v1.12.1
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.44.0
-	go.opentelemetry.io/otel v1.18.0
-	go.opentelemetry.io/otel/trace v1.18.0
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.42.0
+	go.opentelemetry.io/otel v1.16.0
+	go.opentelemetry.io/otel/trace v1.16.0
```

Comment:
@JianyiGao Why are these `go.mod` updates here?

Reply:
These changes come from commit [114dba2](https://github.com/DataDog/dd-trace-go/pull/2314/commits/114dba2dbf5801288de0cc7c1ac61c9bff83710f) merged v2-main

Reply:
@darccio  We checked and removed those, so no updates now in go.mod